lib/elixir/src/elixir_tokenizer.erl

```diff
@@ -609,8 +609,13 @@ handle_strings(T, Line, Column, H, Scope, Tokens) ->
       Token = {Key, {Line, Column - 1, nil}, Unescaped},
       tokenize(Rest, NewLine, NewColumn + 1, Scope, [Token | Tokens]);
     {NewLine, NewColumn, Parts, Rest} ->
-      Token = {string_type(H), {Line, Column - 1, nil}, unescape_tokens(Parts, Scope)},
-      tokenize(Rest, NewLine, NewColumn, Scope, [Token | Tokens])
+      UnescapedToken = unescape_tokens(Parts, Scope),
+      case UnescapedToken of
+        {error, _Msg} = Error -> Error;
```

Comment:
Checking the coding style of this file, it seems right-hand clauses are only kept in the same line if all the clauses are kept as single lines. The same applies to the `unescaped_tokens` function in `lib/elixir/src/elixir_interpolation.erl`

Reply:
thanks! I'm a bit unfamiliar with the style guide, so I trust your judgement! I'll make updates!